plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -454,19 +506,25 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase, MapStyleObserve
     /**
      * Called when an on long press gesture was detected.
      */
-    override fun onLongPress(motionEvent: MotionEvent) {
+    override fun onLongPress(motionEvent: MotionEvent?) {
+      if (motionEvent == null) {
+        return
+      }
       handleLongPressEvent(motionEvent.toScreenCoordinate())
     }
 
     /**
      * Called when an on fling gesture was detected.
      */
     override fun onFling(
-      e1: MotionEvent,
-      e2: MotionEvent,
+      e1: MotionEvent?,
+      e2: MotionEvent?,
       velocityX: Float,
       velocityY: Float
     ): Boolean {
+      if (e1 == null || e2 == null) {
```

Comment:
`e1` is not even used in `handleFlingEvent` - thus we can make `handleFlingEvent` take 3 params instead of 4 and also not check for `e1 == null` then

Reply:
All right